test/Abp.AspNetCore.Tests/App/AppModule.cs

```diff
@@ -27,6 +28,13 @@ public override void PreInitialize()
                 .CreateControllersForAppServices(
                     typeof(AppModule).GetAssembly()
                 );
+
+            Configuration.IocManager.Resolve<IAbpAspNetCoreConfiguration>().RouteConfiguration.Add(routes =>
```

Comment:
Use ``Configuration.Modules.AbpAspNetCore()`` instead of ``Configuration.IocManager.Resolve<IAbpAspNetCoreConfiguration>()``.